src/Tizen.Security.SecureRepository/Tizen.Security.SecureRepository/DataManager.cs

```diff
@@ -55,15 +55,53 @@ static public byte[] Get(string alias, string password)
 
             try
             {
-                Interop.CheckNThrowException(
-                    Interop.CkmcManager.GetData(alias, password, out ptr),
+                CheckNThrowException(
+                    CkmcManager.GetData(alias, password, out ptr),
                     "Failed to get certificate. alias=" + alias);
                 return new SafeRawBufferHandle(ptr).Data;
             }
             finally
             {
                 if (ptr != IntPtr.Zero)
-                    Interop.CkmcTypes.BufferFree(ptr);
+                    CkmcTypes.BufferFree(ptr);
+            }
+        }
+
+        /// <summary>
+        /// Checks for alias existance
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
```

Comment:
Wrong version. It should be ```<since_tizen>8</since_tizen>``` if you want your changes to be merged to the master (API8 branch). Otherwise, please change the branch to API7 (```<since_tizen>7</since_tizen>```).